apps/lockfile-explorer/src/graph/test/graphTestHelpers.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import yaml from 'js-yaml';
+import { FileSystem, NewlineKind } from '@rushstack/node-core-library';
+import {
+  type IJsonLfxGraph,
+  type IJsonLfxWorkspace,
+  lfxGraphSerializer,
+  type LfxGraph
+} from '../../../temp/lfx-shared';
```

Comment:
Importing from something called `temp` is weird. Why not just `lfx-shared`?

Reply:
I've moved it from `lockfile-explorer/temp/lfx-shared` to `lockfile-explorer/build/lfx-shared` and added `build/` to **.gitignore**.  Docusaurus already uses `build` for built outputs, so this is pretty conventional.